mitmproxy/web/app.py

```diff
@@ -303,6 +306,36 @@ def post(self, flow_id):
 
 
 class FlowContent(RequestHandler):
+    def _get_content_view(self, message, viewmode):
```

Comment:
We have more or less the same function three times now (mitmproxy/console/flowview, mitmproxy/builtins/dumper and here). I think this should be centralized.